randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1356,6 +1354,7 @@ Extra - room_id: [49, 59]
   * Open Hatch to Silo Entry/Door to Central Reactor Core
   * Extra - door_idx: (112, 134)
   > Door to Scaffolding Access
+      # Jump on platform, don't fall down: https://youtu.be/tVICpkQOEh0
       Movement (Beginner)
```

Comment:
The platform was created to prevent nettori from being dangerous. In vanilla, you can traverse the path by using the vines. Since the vines dissapear after nettori, it meant that killing nettori would kill the path.

Beginner movement is due to the fact that if you somehow manage to miss the jump and fall down, that you're softlocked.